src/app/loginComponents/accounts/external/accounts.component.ts

```diff
@@ -28,23 +28,27 @@ import { UserQuery } from '../../../shared/user/user.query';
 import { UserService } from '../../../shared/user/user.service';
 import { TokenUser } from './../../../shared/swagger/model/tokenUser';
 import { AccountsService } from './accounts.service';
+import { User } from '/Users/elin/dockstore-ui2/src/app/shared/swagger/model/user';
```

Comment:
this one

Reply:
Thank you, I just made the change. I must've forgot to change it.